recipes/juliacall/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.4.3" %}
+
+package:
+  name: juliacall
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/juliacall/juliacall-{{ version }}.tar.gz
+  sha256: ff81c152831775bbbb26b7752a7825d1de2fd6c154f71ffc22338996b4faac07
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
```

Comment:
can't you import `juliacall` for test?

Reply:
Note that conda-build includes an `imports` keyword: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports